src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -225,6 +226,11 @@ trait ScalaSettings extends AbsScalaSettings
   val exposeEmptyPackage = BooleanSetting ("-Yexpose-empty-package", "Internal only: expose the empty package.").internalOnly()
   val Ydelambdafy        = ChoiceSetting  ("-Ydelambdafy", "strategy", "Strategy used for translating lambdas into JVM code.", List("inline", "method"), "method")
 
+  val YaddBackendThreads = IntSetting   ("-YaddBackendThreads", "maximum additional threads for backend", 0, Some((0,16)), (x: String) => None )
+  val YmaxQueue = IntSetting   ("-YmaxQueue", "backend additional threads worker queue size", 0, Some((0,1000)), (x: String) => None )
+  val YjarCompressionLevel = IntSetting("-YjarCompressionLevel", "compression level to use when writing jar files",
+    Deflater.DEFAULT_COMPRESSION, Some((Deflater.DEFAULT_COMPRESSION,Deflater.BEST_COMPRESSION)), (x: String) => None)
```

Comment:
By convention, compiler options are hyphenated rather than camel cased.

I think we can improve the name, too. How about `-Ybackend-parallelism`, which default to 1 (for now).

Reply:
-YaddBackendThreads -> -Ybackend-parallelism
minimum 1, so its the total threads not the additional threads
-YmaxQueue -> -Ybackend-worker-queue?
-YjarCompressionLevel -> Yjar-compression-level?
@retronym would that suit you?

Reply:
assumed this is OK and added a commit for this